sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -404,6 +401,25 @@ private void cancelScheduledNetworkUpdate() {
         }
     }
 
+    private void cancelAllSubscriptions() {
+        ArrayList<SubscriptionHandle> handles = new ArrayList<>(subscriptions);
+        for (SubscriptionHandle handle : handles) {
+            handle.unsubscribe();
+        }
+    }
+
+    void trackSubscription(SubscriptionHandle subscriptionHandle) {
+        if (!subscriptions.contains(subscriptionHandle)) {
+            subscriptions.add(subscriptionHandle);
+        }
```

Comment:
Redundant to check contains before adding with a set. But since switching to a list can just add.